jib-core/src/test/java/com/google/cloud/tools/jib/frontend/CredentialRetrieverFactoryTest.java

```diff
@@ -44,118 +47,189 @@
 
   private static final Credential FAKE_CREDENTIALS = Credential.from("username", "password");
 
-  /**
-   * Returns a {@link DockerCredentialHelperFactory} that checks given parameters upon creating a
-   * {@link DockerCredentialHelper} instance.
-   *
-   * @param expectedRegistry the expected registry given to the factory
-   * @param expectedCredentialHelper the expected credential helper path given to the factory
-   * @param returnedCredentialHelper the mock credential helper to return
-   * @return a new {@link DockerCredentialHelperFactory}
-   */
-  private static DockerCredentialHelperFactory getTestFactory(
-      String expectedRegistry,
-      Path expectedCredentialHelper,
-      DockerCredentialHelper returnedCredentialHelper) {
-    return (registry, credentialHelper) -> {
-      Assert.assertEquals(expectedRegistry, registry);
-      Assert.assertEquals(expectedCredentialHelper, credentialHelper);
-      return returnedCredentialHelper;
-    };
-  }
-
-  @Mock private Consumer<LogEvent> mockLogger;
-  @Mock private DockerCredentialHelper mockDockerCredentialHelper;
-  @Mock private DockerConfigCredentialRetriever mockDockerConfigCredentialRetriever;
-
-  /** A {@link DockerCredentialHelper} that throws {@link CredentialHelperNotFoundException}. */
-  @Mock private DockerCredentialHelper mockNonexistentDockerCredentialHelper;
-
-  @Mock private CredentialHelperNotFoundException mockCredentialHelperNotFoundException;
+  @Mock private Consumer<LogEvent> logger;
+  @Mock private DockerCredentialHelper dockerCredentialHelper;
+  @Mock private DockerCredentialHelperFactory dockerCredentialHelperFactory;
+  @Mock private GoogleCredentials googleCredentials;
```

Comment:
Possible to back these `mock` prefix changes out too?

Reply:
Oops, missed this class.